ref/net46/Microsoft.Build.Tasks.Core/Microsoft.Build.Tasks.Core.cs

```diff
@@ -1163,10 +1165,11 @@ public partial class WinMDExp : Microsoft.Build.Tasks.ToolTaskExtension
         protected override System.Text.Encoding StandardOutputEncoding { get { throw null; } }
         protected override string ToolName { get { throw null; } }
         public bool TreatWarningsAsErrors { get { throw null; } set { } }
+        protected override bool UseNewLineSeparatorInResponseFile { get { throw null; } }
         public bool UTF8Output { get { throw null; } set { } }
         [Microsoft.Build.Framework.RequiredAttribute]
         public string WinMDModule { get { throw null; } set { } }
-        protected internal override void AddCommandLineCommands(Microsoft.Build.Tasks.CommandLineBuilderExtension commandLine) { }
+        protected internal override void AddResponseFileCommands(Microsoft.Build.Tasks.CommandLineBuilderExtension commandLine) { }
```

Comment:
Can you leave the old method there, in case somebody is extending this Task? I searched on Index and first 7 pages of Github results though and no hit.